common/common.go

```diff
@@ -249,6 +249,15 @@ func RoundFloat(x float64, prec int) float64 {
 	return rounder / pow
 }
 
+// CountInt counts how many decimals in an integer and returns the count
```

Comment:
Do you mean to be counting digits? Or does golang int support decimal places?

Reply:
Yes I do, please request change.